MPF/ViewModels/MainViewModel.cs

```diff
@@ -570,6 +570,53 @@ public async void InitializeUIValues(bool removeEventHandlers, bool rescanDrives
             App.Instance.StartStopButton.IsEnabled = ShouldEnableDumpingButton();
         }
 
+        /// <summary>
+        /// Performs a fast update of the output path while skipping disc checks
+        /// </summary>
+        /// <param name="removeEventHandlers">Whether event handlers need to be removed first</param>
+        private void FastUpdateLabel(bool removeEventHandlers)
+        {
+            // Disable the dumping button
+            App.Instance.StartStopButton.IsEnabled = false;
+
+            // Safely uncheck the parameters box, just in case
+            if (App.Instance.EnableParametersCheckBox.IsChecked == true)
+            {
+                App.Instance.EnableParametersCheckBox.Checked -= EnableParametersCheckBoxClick;
+                App.Instance.EnableParametersCheckBox.IsChecked = false;
+                App.Instance.ParametersTextBox.IsEnabled = false;
+                App.Instance.EnableParametersCheckBox.Checked += EnableParametersCheckBoxClick;
+            }
+
+            // Set the UI color scheme according to the options
+            if (App.Options.EnableDarkMode)
+                EnableDarkMode();
+            else
+                DisableDarkMode();
+
+            // Force the UI to reload after applying the theme
+            App.Instance.UpdateLayout();
+
+            // Remove event handlers to ensure ordering
+            if (removeEventHandlers)
+                DisableEventHandlers();
+
+            // Refresh the drive info
+            Drive drive = App.Instance.DriveLetterComboBox.SelectedItem as Drive;
+            drive.RefreshDrive();
+
+            // Set the initial environment and UI values
+            Env = DetermineEnvironment();
+            GetOutputNames(true, true);
```

Comment:
Setting `driveChanged` only should be sufficient. Adding the second flag shouldn't be needed as `driveChanged` does nothing but determine that the previous path may have already existed.